<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bspTestForm</class>
 <widget class="QWidget" name="bspTestForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>825</width>
    <height>673</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="cpldresetpushButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>CPLD复位</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>611</width>
     <height>481</height>
    </rect>
   </property>
   <property name="title">
    <string>PWM模块</string>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>591</width>
      <height>441</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>单相开管</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>双脉冲</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>481</width>
        <height>151</height>
       </rect>
      </property>
      <property name="title">
       <string>参数设置</string>
      </property>
      <widget class="QLineEdit" name="firstPuselineEdit">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>30</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>10</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>第一个脉冲宽度：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>30</y>
         <width>31</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>us</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>60</y>
         <width>31</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>us</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>第二个脉冲宽度：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="secondPuselineEdit">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>60</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>2</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>90</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>脉冲的间隔宽度：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="betweenPuselineEdit">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>90</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>2</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>90</y>
         <width>31</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>us</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>481</width>
        <height>121</height>
       </rect>
      </property>
      <property name="title">
       <string>相桥选择</string>
      </property>
      <widget class="QCheckBox" name="checkBoxAUp">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>A相上管</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBoxADown">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>60</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>A相下管</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBoxBUp">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>30</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>B相上管</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBoxBDown">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>60</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>B相下管</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBoxCUp">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>30</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>C相上管</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBoxCDown">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>60</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>C相下管</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="startpushButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>330</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>触发</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>321</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>通信</string>
   </property>
   <widget class="QPushButton" name="connectpushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>连接</string>
    </property>
   </widget>
   <widget class="QPushButton" name="disconnectpushButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>断开</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
